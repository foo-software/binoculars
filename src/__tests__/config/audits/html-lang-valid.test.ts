import auditHtmlLangValid from '../../../config/audits/html-lang-valid';

describe('html-lang-valid audit', () => {
  it('should match snapshot', () => {
    expect(auditHtmlLangValid('en')).toMatchSnapshot();
  });
});
