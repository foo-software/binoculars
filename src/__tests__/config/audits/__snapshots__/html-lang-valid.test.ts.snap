// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`html-lang-valid audit should match snapshot 1`] = `
Object {
  "description": "Specifying a valid [BCP 47 language](https://www.w3.org/International/questions/qa-choosing-language-tags#question) helps screen readers announce text properly. [Learn more](https://web.dev/html-lang-valid/).",
  "title": "\`<html>\` element has a valid value for its \`[lang]\` attribute",
}
`;
